<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pB_ajouter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_supprimer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse MAC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_adresse">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>NN:NN:NN:NN:NN:NN</string>
   </property>
   <property name="text">
    <string>:::::</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Propriétaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Appareil</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_appareil">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbB_proprietaire">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_activer_ajouter">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_supprimer_annuler">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_proprietaire">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tW_bd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>360</width>
     <height>191</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Appareil</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adresse</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Propriétaire</string>
    </property>
   </column>
  </widget>
  <zorder>lE_proprietaire</zorder>
  <zorder>pB_ajouter</zorder>
  <zorder>pB_supprimer</zorder>
  <zorder>label</zorder>
  <zorder>lE_adresse</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lE_appareil</zorder>
  <zorder>cbB_proprietaire</zorder>
  <zorder>pB_activer_ajouter</zorder>
  <zorder>pB_supprimer_annuler</zorder>
  <zorder>tW_bd</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
